src/components/Class/index.js

```diff
@@ -98,7 +98,9 @@ function Class({ classToEdit }) {
 
   const editClass = (payload) => {
     if (classToEdit.type === "cohort") {
-      delete payload.frequency;
+      if (indicator === false) {
+        delete payload.frequency;
```

Comment:
@Poonam-Singh-Bagh Are you also remove other cohort related keys from the payload like `occurrence`, `on_days` or `until`

Reply:
No. Because we are not sending these fields as payload when the user wants to edit only one class.